connector/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceProvider.scala

```diff
@@ -357,6 +358,29 @@ private[kafka010] class KafkaSourceProvider extends DataSourceRegister
     validateGeneralOptions(params)
   }
 
+  private def translateEventStreamProperties(params: CaseInsensitiveMap[String])
+  : CaseInsensitiveMap[String] = {
+    if (params.contains(EVENTSTREAM_NAME_OPTION_KEY) ||
+      params.contains(EVENTSTREAM_ARTIFACT_ID_OPTION_KEY) ||
+      params.contains(EVENTSTREAM_CONSUMER_GROUP_OPTION_KEY)) {
+      validateEventStreamOptions(params)
+      return CaseInsensitiveMap(KafkaOptionsUtils.buildKafkaOptionsFromSparkConfig(params));
```

Comment:
Method name should be "EventStreamUtils", KafkaOptionsUtils is a generic name which is more suitable for open source kafka connector utils